core/node/da_clients/src/eigen/eigenda-integration.md

```diff
@@ -46,6 +46,14 @@ da_client:
       authenticated: false
 ```
 
+Note: `authenticated` dispersal requires setting up a private key in `etc/env/file_based/secrets.yaml`:
+
+```yaml
+da:
+  eigen:
+    private_key: '<your_private_key>'
+```
+
```

Comment:
Even if authenticated is set to false, we still need the secret

Reply:
You are right, fixed it